### DISCUSSION TITLE
Passing current model id to a livewire component

### DISCUSSION THREAD

**Discussion by Causeofhell** [filamentphp/filament#13095] [Help]
### Package

Infolist builder

### Package Version

v3.2

### How can we help you?

How can i get the actual model in a infolist view and pass it to live wire component?
Livewire::make(RepaymentSchedulesTable::class, ['recordId' => '1']) 
i want to get the current model id and pass it i am not using a custom page so i want to use it in a infolist view
is possible to get it ?

**Comment by leandrocfe** [comment]
`$getRecord()` ?

**Comment by stmunker** [comment]
The current model is available as `$record` in your livewire component.
`public Model $record;`

See here [https://filamentphp.com/docs/3.x/infolists/advanced#accessing-the-current-record-in-the-livewire-component](https://filamentphp.com/docs/3.x/infolists/advanced#accessing-the-current-record-in-the-livewire-component)

**Reply by Causeofhell** [reply]
nah is not possible because i am working with a local values i have a form i got the information and i want to populate a table with that array

**Reply by leandrocfe** [reply]
Could you share the whole code that you are using?

**Reply by stmunker** [reply]
In the resource class:
```
public static function infolist(Infolist $infolist): Infolist
{
    return $infolist
        ->schema([
            Infolists\Components\Livewire::make(CustomerOverview::class),
        ]);
}
```

In the Livewire class
```
use Illuminate\Database\Eloquent\Model

class CustomerOverview extends Component
{
    public Model $record;

    public function render()
    {
        return view('livewire.customer-overview');
    }
}
```

In the Livewire Blade file:
```
<div>
    {{ $record->firstname }}
    {{ $record->lastname }}
</div>
```
